hocr-split

```diff
@@ -30,23 +21,23 @@ pattern = sys.argv[2]
 assert re.search('%[0-9]*d',pattern)
 
 stream = open(sys.argv[1])
-doc = HtmlLib.Reader().fromString(stream.read())
+doc = html.fromstring(stream.read())
 
-template = doc.cloneNode(1)
-pages = xquery("//*[@class='ocr_page']",template)
+template = copy.deepcopy(doc)
+pages = doc.xpath("//*[@class='ocr_page']")
 assert pages!=[]
-container = pages[0].parentNode
+container = pages[0].getparent()
 
 def write_template_with_page(new_page,index):
-    pages = xquery("//*[@class='ocr_page']",template)
-    for page in pages: container.removeChild(page)
-    container.appendChild(new_page)
-    stream = open(pattern % index,"w")
-    xml.dom.ext.PrettyPrint(doc,stream)
+    pages = container.xpath("//*[@class='ocr_page']")
```

Comment:
@kba Did you switch here from `template` to `container` (if compared with the original) on purpose, or is this a typo?